labgrid/driver/fastbootdriver.py

```diff
@@ -54,7 +57,9 @@ def boot(self, filename):
 
     @Driver.check_active
     @step(args=['partition', 'filename'])
-    def flash(self, partition, filename):
+    def flash(self, partition, filename=None):
```

Comment:
The other drivers support images don't have the complexity of multiple partitions.
For fastboot, there are no standardized partition names a far as I know.
So for flash(), making the filename configurable but not the partition seems strange.

Maybe something like this would be more useful:
```
…
AndroidFastbootDriver:
  boot_image: "kernel" # referring to a path under images
  flash_images:
    nand: "nand-image"
    nor: "nor-image"
…
```

Then the attribute would need to validate as a dict instance.
A flashall method could then iterate over all images, and flash with only a partition could lookup the correct image.

What do you think?

Reply:
Yes, I think that would be better. I don't know when I will have the time to implement that.

Reply:
Done.